test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -400,3 +400,10 @@ test('should not create proxy url for invalid url (GH-778)', function () {
         strictEqual(linkVal, nativeLinkVal);
     }
 });
+
+test("Should not use 'autocomplete' getter if an object isn't DOM Element (GH-942)", function () {
```

Comment:
test("Should not use the 'autocomplete' getter if an object isn't a DOM element  (GH-942)", function () {

// NOTE: This code checks the PropertyAccessorsInstrumentation.attach(window).autocomplete condition